ros_ign_gazebo/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ target_link_libraries(create
   ignition-transport${IGN_TRANSPORT_VER}::core
 )
 
-add_library(Stopwatch src/Stopwatch.cpp)
+add_library(Stopwatch SHARED src/Stopwatch.cpp)
```

Comment:
what do you think about calling this library `ros_ign_gazebo` instead of `Stopwatch`?

Reply:
It's alright with me. You mean this pattern?

```
add_library(${PROJECT_NAME} SHARED src/Stopwatch.cpp)

install(
  TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}
  ARCHIVE DESTINATION lib
  LIBRARY DESTINATION lib
  RUNTIME DESTINATION bin
)
```

Reply:
The counterpoint to this is having smaller libraries broken up with lighter dependency footprints.  I think that this has enough utility to live in a "common library", though.